Makefile

```diff
@@ -330,7 +330,7 @@ i18n-extract: ## Extract strings for translation from the source code
 	$(GOPATH)/bin/mmgotool i18n extract
 
 store-mocks: ## Creates mock files.
-	env GO111MODULE=off go get -u github.com/vektra/mockery/...
+	env GO111MODULE=off go get -u github.com/vektra/mockery/cmd/mockery
```

Comment:
What's the reason for this change? It doesn't seem related to the rest of the PR.

Reply:
Done